README.md

```diff
@@ -157,4 +157,7 @@ Feel free to check the [issues page](https://github.com/eyobyoseph/Eyob-portfoli
 
 This project is [MIT](./LICENSE) licensed.
 
+NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+
```

Comment:
- [ ] Great job adding the `MIT.md` file, but when I click the MIT link it is giving a `404` error. I think this is happening because the file is named `MIT.md` while the link is referencing `LICENSE`. Kindly rename one of them to fix the issue. 🧐️ Please also remove this NOTE from the README file as it is meant to be a guide on what to do. 

<img src="https://user-images.githubusercontent.com/17243325/220624280-22a274d2-4858-4598-abc8-81ef87183068.png" width="500">

Reply:
It is adjusted according to the advise given by code reviewer